### ISSUE TITLE
Normalize based only on training data

### ISSUE THREAD

**Issue by yanii** [mansdell/AO-PWC#2]
At the moment the mean and standard deviation are calculated over the entire training and validation dataset. Should only be calculated based on training set. This necessarily requires us to have a fixed training and validation set, or to normalize at runtime. Fixed training/test sets are better for repeatability however.

**Comment by tom-roddick** [comment]
Good point! I think at some point we should definitely move to a fixed train/val split, maybe once Becky comes back with the updated dataset. It might actually be worth leaving the dataset un-normalised and then normalising inside the model instead?

**Comment by yanii** [comment]
If it's just a matter of normalizing with a fixed scalar mean/stddev from the training set then yeah, just do it in the training script. Have to make sure we only calculate based on training set then too.

Right now my training and val RMS error are the exact same at the end 😕

Btw nice bonus of the new smaller model - way faster to train and uses less GPU memory.

Yani
